scripts/scil_compute_rish_from_sh.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Script to compute the RISH features from an SH coefficients image.
```

Comment:
Please add some meat here. 
- Add a reference.
- What is a RISH. Define it. in words and in principle